File: 283-move-zeroes/move-zeroes.py
class Solution:
    def moveZeroes(self, nums):
        nmbr = 0

        for i in range(len(nums)):
            if nums[i] != 0:
                temp = nums[i]
                nums[i] = nums[nmbr]
                nums[nmbr] = temp
                nmbr += 1

File: 283-move-zeroes/move-zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
        int nmbr=0;
        int temp;
        for(int i=0; i<nums.length; i++){
            if(nums[i] != 0){
                temp=nums[i];
                nums[i]=nums[nmbr];
                nums[nmbr]=temp;
                nmbr++;
            }
        }
    }
}